comps/comps-katello-server-rhel6.xml

```diff
@@ -29,6 +29,7 @@
        <packagereq type="default">tfm-rubygem-foreman_pipeline</packagereq>
        <packagereq type="default">tfm-rubygem-hammer_cli_katello</packagereq>
        <packagereq type="default">tfm-rubygem-hammer_cli_import</packagereq>
+       <packagereq type="default">client-boostrap</packagereq>
```

Comment:
yes, it is going to land in /var/lib/www/html/pub and clients will wget it, similar to the consumer-certs RPM. We need it accessible prior to registration in an unprotected public location so you can 'bootstrap' and get registered and then get content.